concent_api/concent_api/middleware.py

```diff
@@ -55,7 +56,10 @@ def __call__(self, request):
 def determine_return_type(request_meta):
     try:
         preferred_types = ['application/json', 'text/html']
```

Comment:
We have a bug here:

From [best_match()](https://github.com/dbtsai/python-mimeparse/blob/master/mimeparse.py#L155-L184) docstring:
> (...) The value of 'supported'
> is a list of mime-types. The list of supported mime-types should be sorted
> in order of increasing desirability, in case of a situation where there is
> a tie.

So `'application/json'` should be last if we want it to be chosen when the quality values are equal but we have it first.

Reply:
:+1: done, and yes, the original issue was caused by this.